serialDecoder.py

```diff
@@ -303,9 +303,9 @@ def mainLoop(args):
     	global grapher 
 	grapher = grapher([0])
 	if args.csv:
-	    print args.port[0] + ','
+	    print((args.port[0] + ','))
```

Comment:
All of the lines like this are technically correct changes, but have an unnecessary set of parentheses because ```print("first"+"second")``` works correctly.